### ISSUE TITLE
Logos don't have contrast with new header background

### ISSUE THREAD

**Issue by Felix-Icaza** [midudev/kings-league-project#248]
The new design of the header looks really nice 🎉, but the logos of the Los Troncos FC and Kunisports don't have good contrast with the background 😔.

![Screenshot_2023-01-08_08-48-39](https://user-images.githubusercontent.com/38303370/211202812-8212ac68-2d06-46e1-95e8-054829b934a3.png)

**Comment by magdaniele** [comment]
yes I feel the same, I was trying to see how it likes whit white but I'm not convinced yet

look:
![image](https://user-images.githubusercontent.com/25647270/211203360-2a2e2025-9b9e-4215-a1a8-a90a1c6a8ae6.png)

**Comment by Felix-Icaza** [comment]
We can propose which colors to use.

In the following screenshot I use the same colors of Kings League logo but in reverse (yellow background and black border).

![Screenshot_2023-01-08_09-15-51](https://user-images.githubusercontent.com/38303370/211204385-6533bc60-7397-4479-bfc9-b257e87637d3.png)

**Comment by magdaniele** [comment]
> We can propose which colors to use.
> 
> In the following screenshot I use the same colors of Kings League logo but in reverse (yellow background and black border).
> 
> ![Screenshot_2023-01-08_09-15-51](https://user-images.githubusercontent.com/38303370/211204385-6533bc60-7397-4479-bfc9-b257e87637d3.png)

I like it, modify it and let's see what midu says.

**Comment by Felix-Icaza** [comment]
Thanks @magdaniele, I will send a PR.
